applications/leaderboard/README.md

```diff
@@ -0,0 +1,68 @@
+# .NET Cloud Spanner Sample Leaderboard Application
```

Comment:
Could you add a brief note about why there are step1, step2, etc. directories?

Reply:
Done.